direct/io-hbase-bindings/src/main/java/cz/o2/proxima/direct/hbase/HBaseLogObservable.java

```diff
@@ -196,4 +196,20 @@ private Filter toFilter(List<AttributeDescriptor<?>> attributes) {
 
     return attrFilter;
   }
+
+  @Override
+  public int hashCode() {
+    return 69;
```

Comment:
Minor: this is not a prime, prime would have been better for this.

Reply:
```suggestion
    return 73;
```

Reply:
fixed